ci/docker-db.sh

```diff
@@ -1,59 +1,76 @@
 #!/usr/bin/env bash
-# source this file to setup postgres and mysql
-# for local testing (as similar as possible to docker)
+# This scripts starts a postgres or mysql database server, waits for it, and
+# setup setups users to be used for local testing.
 
 set -eu
 
-export MYSQL_HOST=127.0.0.1
+# These environment variables influence the mysql and psql clients
```

Comment:
In this script, unlike init-db, this comment is not quite accurate. The ports influence the server, and must be set. Like `init-db`, the HOST args appear to be unused and can be omitted.